react-native-scripts/template/README.md

```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint and Prettier
```

Comment:
can we split this up so we have an ESLint section and a Prettier section? in CRA the section is called "Formatting Code Automatically" https://github.com/facebookincubator/create-react-app/blob/b555192b43876c423a81f0cd699f3f4a94aa7441/packages/react-scripts/template/README.md#formatting-code-automatically